packages/config/config/devices/0x0090/893.json

```diff
@@ -0,0 +1,175 @@
+{
+	"manufacturer": "Spectrum Brands",
+	"manufacturerId": "0x0090",
+	"label": "893",
+	"description": "10 Button Z-Wave Plus v2 Deadbolt",
+	"devices": [
+		{
+			"productType": "0x0811",
+			"productId": "0x13a8",
+			"zwaveAllianceId": 4184
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Status LED Control",
+			"description": "The configuration parameter 1 is a read/write parameter with a one-byte field, used to control the status led. if set to on, the status LED will show the status of the latch every 6 seconds. if set to off, the status LED will not be shown",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"writeOnly": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Turns status LED off",
+					"value": 0
+				},
+				{
+					"label": "Turns status LED on",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Sound Control",
+			"description": "The configuration parameter 2 is a read/write parameter with a one-byte field, used to control the lock’s buzzer. if set to on, the buzzer is enabled and will sound during normal events. if set to off, the buzzer is disabled, and the buzzer sound will not occur",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"writeOnly": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Turns the sound (buzzer) off",
+					"value": 0
+				},
+				{
+					"label": "Turns the sound (buzzer) on",
+					"value": 1
+				}
+			]
```

Comment:
```suggestion
			"options": [
				{
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Enable",
					"value": 1
				}
			]
```

Reply:
...and so on. See https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide for all rules